pkg/cmd/roachtest/test_runner.go

```diff
@@ -605,7 +605,10 @@ func (r *testRunner) runWorker(
 			qp.Release(alloc)
 		}
 	}()
-
+	// cluster destroy can be done concurrently.
```

Comment:
Nit: a single line comment should suffice. What part of this code isn't self-documenting? Arguably, only the `defer` statement may need a supporting comment.